### PR TITLE
Eb 132 docker

### PR DESCRIPTION
**Pull request by MetaEMK**
Please check if you can work with the installation Readme
please use git clone -b EB-132_Docker .... instead

### CODE REVIEW

File: README.md
Comment (J-C-V) [inline README.md:1]: @MetaEMK Info:

- Removed local installation not worth it (Needs Supabase registration and more...)
- Rephrased the whole getting started section

File: docker/README.md
Comment (J-C-V) [inline docker/README.md:1]: @MetaEMK Info:

- Restructured some parts
- added comments

**Review by J-C-V — CHANGES_REQUESTED** [review]

File: Dockerfile
[the remark this answers is not in the document]
Comment (MetaEMK) [reply Dockerfile:1]: fixed

**Review by J-C-V — APPROVED** [review]
LGTM

**Pull request merged**